common/src/testFixtures/kotlin/org/onflow/flow/common/test/AbstractFlowEmulatorExtension.kt

```diff
@@ -212,6 +231,12 @@ abstract class AbstractFlowEmulatorExtension : BeforeEachCallback, TestExecution
                 )
             }
 
+            // Validate key size
+            val publicKey = keyPair.public.hex
+            if (annotation.signAlgo == SignatureAlgorithm.ECDSA_P256 && publicKey.length != 128) {
```

Comment:
- If there is no reason to restrict the algo to `ECDSA_P256`, we could remove the check. It should be possible to add account keys for all ecdsa algos.
- the length check for `128` could be removed too, because `generateKeyPair` and `decodePublicKey` guarantee the length is correct

Reply:
Good catch, I added this while debugging and forgot to remove. Thanks!